hatch/src/cli/commands/new.rs

```diff
@@ -37,6 +38,20 @@ impl<'new> New {
       ProjectKind::Library(LibraryKind::Shared)
     }
   }
+
+  fn get_includes(&self, args: &ArgMatches<'new>) -> Vec<String> {
+
+    let re = Regex::new(r"https?://(www\.)?[-a-zA-Z0-9@:%._\+~#=]{2,256}\.[a-z]{2,4}\b([-a-zA-Z0-9@:%_\+.~#?&//=]*)").unwrap();
```

Comment:
What are we testing for here with regex?

Reply:
Valid url regex. I can add a comment for clarification.

Reply:
Is it only checking for `https://` ?

Reply:
Can we do this without regex? It feels very overkill.

Reply:
Going to remove this.